workers/docker-worker/src/features/chain_of_trust.js

```diff
@@ -27,11 +30,13 @@ class ChainOfTrust {
     this.ed25519Key = Buffer.from(await new Promise((accept, reject) =>
       fs.readFile(task.runtime.ed25519SigningKeyLocation, 'ascii', (err, data) => err ? reject(err) : accept(data))), 'base64');
 
-    this.file = new temporary.File();
-    debug(`created temporary file: ${this.file.path}`);
+    this.tmpDir = mkdtempSync(`${tmpdir()}${sep}`);
```

Comment:
it looks like other parts of this feature are using async methods, maybe it's worth replacing this only sync method with `await mkdtemp`